### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by singularitti**
It was added in https://github.com/Homebrew/homebrew-cask/pull/95418. I will keep an eye on this app & update it if you have a new release.

### CODE REVIEW

**Comment by DenBeke** [comment]
Neat! Thanks for this contribution!

How to keep the Homebrew version up-to-date with the releases from this Github repo?

File: README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 Latest version can be found on [releases page](https://github.com/DenBeke/YouTube-Downloader-for-macOS/releases).
 
+If you have [Homebrew](https://brew.sh/) installed, try run the following in your terminal:
```
Comment (DenBeke) [inline README.md:15]: Can we word this more direct. Like:

> You can also install it with Homebrew:
> ```shell
> brew install --cask youtube-downloader
> ```

Comment (singularitti) [reply README.md:15]: Sure!

**Comment by singularitti** [comment]
> Neat! Thanks for this contribution!
> 
> How to keep the Homebrew version up-to-date with the releases from this Github repo?

Well, GitHub will notify me if you have a new release. And I will update it by myself.

**Comment by DenBeke** [comment]
Great!

Thank you for contributing, @singularitti !

**Pull request merged**